server/services/utils/conditionsFormatter.ts

```diff
@@ -54,13 +54,18 @@ const dateFields = Object.entries({
   appointmentDateInDrugsSection: {
     day: 'appointmentDayInDrugsSection',
     month: 'appointmentMonthInDrugsSection',
-    year: 'appointmentMonthInDrugsSection',
```

Comment:
this was a recently introduced bug that prevented any of these dates being persisted